www/css/main.diary.css

```diff
@@ -46,8 +46,8 @@
 /* ----------- iPhone 5 and 5S ----------- */
 
 /* Portrait and Landscape */
-@media only screen 
```

Comment:
@trevor-wu can you please set your editor to not modify whitespace? It messes up the git blame.
As we discussed earlier "Hide whitespace changes" just hides the changes in the review; they still make it through to the git history.